samples/noxfile_config.py

```diff
@@ -1,20 +1,15 @@
 TEST_CONFIG_OVERRIDE = {
-    # You can opt out from the test for specific Python versions.
-    "ignored_versions": ["2.7"],
-    # Old samples are opted out of enforcing Python type hints
-    # All new samples should feature them
-    "enforce_type_hints": True,
```

Comment:
The comment above does not reflect the current state as the existing samples are not opted out. I've deleted this line. If we want to add this setting, it should be done in a separate PR.